flink-connectors/flink-connector-filesystem/src/test/java/org/apache/flink/streaming/connectors/fs/RollingSinkFaultToleranceITCase.java

```diff
@@ -77,8 +77,8 @@
 
 	private static String outPath;
 
-	@BeforeClass
-	public static void createHDFS() throws IOException {
+	@Before
```

Comment:
This is because its parent `StreamFaultToleranceTestBase` introduced parameterized test for two different failover strategies, for which I want to verify whether streaming programs could still be fault tolerant with `RestartPipelinedRegionStrategy`. However, files on hdfs left within `RollingSinkFaultToleranceITCase` after this test has run once would cause it failed in the new run with another strategy. That's why I have to modify this test.
I think after this PR, [FLINK-10713](https://issues.apache.org/jira/browse/FLINK-10713) should also add `RestartIndividualStrategy` into `StreamFaultToleranceTestBase`.